packages/opencensus-propagation-binaryformat/src/ext-types.d.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare module 'hex2dec' {
```

Comment:
What's the role of this module?

Reply:
The type definitions for ```hex2dec (@types/hex2dec)``` is not yet available, so added this module to declare ```hex2dec``` module.

Compile error without this module:
```
TS7016: Could not find a declaration file for module 'hex2dec'. 'opencensus-node/packages/opencensus-propagation-binaryformat/node_modules/hex2dec/index.js' implicitly has an 'any' type.
  Try `npm install @types/hex2dec` if it exists or add a new declaration (.d.ts) file containing `declare module 'hex2dec';
```

Reply:
OK, makes sense.